x-pack/plugins/security/public/session/session_timeout.ts

```diff
@@ -108,8 +104,14 @@ export class SessionTimeout {
   /**
    * Event handler that receives session information from other browser tabs.
    */
-  private handleChannelMessage = (message: SessionState) => {
-    this.sessionState$.next(message);
+  private handleChannelMessage = (messageEvent: MessageEvent<any>) => {
+    if (this.isSessionState(messageEvent.data)) {
+      this.sessionState$.next(messageEvent.data);
+    }
+  };
+
+  private isSessionState = (data: any): data is SessionState => {
+    return Object.hasOwn(data ?? {}, 'canBeExtended');
```

Comment:
I was curious how we might be able to avoid this additional explicit `any` so I tried a few things --

This seems to make the TS checker happy:

```suggestion
  private isSessionState = (data: unknown): data is SessionState => {
    return typeof data === 'object' && Object.hasOwn(data ?? {}, 'canBeExtended');
```

Reply:
I like this, thanks!